style/samples/ObjectiveC.m

```diff
@@ -49,8 +46,8 @@ - (id)initWithCoder:(NSCoder *)aDecoder
 - (void)shuffleCards
 {
     // Objective-C literals are your friend
-    NSDictionary *themeColors = @{ kRedColor : [UIColor redColor], kBlueColor : [UIColor blueColor] };
-    NSArray *robots = @[ @"Ralph", @"Bender", @"The Iron Giant" ];
+    NSDictionary *themeColors = @{redColor: [UIColor redColor], blueColor: [UIColor blueColor]};
```

Comment:
I prefer keeping `kRedColor` even though we don't declare those constants and it wouldn't compile. `kBlueColor` screams "constant" (and I think that guideline is in the guides) whereas `blueColor` screams "I did something crazy with `copyWithZone:` to use this object as a key."